client/src/style/header.css

```diff
@@ -0,0 +1,32 @@
+header{
+    
+    background-image: 
+    linear-gradient(
+            rgba(3, 2, 2, 0.541), 
+            rgba(5, 4, 4, 0.589)
+          ),url("../images/pplmakeglasgow.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    min-height: 300px;
+    background-position: right bottom, left top;
+    background-size: 100% 600px;
+    color:white;
+    display: flex!important;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    
+    
+}
+
+@media (max-width: 600px) { 
```

Comment:
you should have mobile view as the default, and and desktop as the exception (mobile-first). So have the default to be the case for Mobile, then override it with `@media (min-width: 600 ..` to change for desktop - not the other way around

Reply:
Also use the Glasgow background for both cases - rmeove the coffee image